docs/Glossary/index.md

```diff
@@ -0,0 +1,143 @@
+# Glossary
+
+This glossary provides definitions and explanations of commonly used terms, abbreviations, and phrases organized by the relevant application or function.
+
+---
+
+## Media Management Applications
+
+**Radarr**, **Sonarr**, **Lidarr**, and **Prowlarr** are applications designed to help with management of your media libraries and automation. This group of applications is commonly referenced collectively as the "arrs".
+
+| Application                          | Content Type       |
+| ------------------------------------ | ------------------ |
+| [**Radarr**](../Radarr/index.md)     | Movies             |
+| [**Sonarr**](../Sonarr/index.md)     | TV Shows/Series    |
+| [**Lidarr**](../Lidarr/index.md)     | Music              |
+| [**Prowlarr**](../Prowlarr/index.md) | Indexer Management |
+
+### Custom Format
+
+A feature that uses pattern matching on a release, based on the information implied through the release name on the corresponding indexer during a search. Users can customize patterns to match a Custom Format, and Custom Formats can then be assigned scores individually in each [Quality Profile](#quality-profile). After the release has been imported, the Custom Format score will be updated to match the properties of the actual media file, rather than strictly by its original naming alone.
+
+### Quality
+
+Refers to the specific release quality that describes both the source and, optionally, the resolution. Examples of qualities include **Bluray-2160p** and **Remux-1080p**, though many others exist.
+
+### Quality Definitions
+
+Defines the minimum, preferred, and maximum file sizes for any given [Quality](#quality). Sizes are measured in megabytes per minute.
+
+### Quality Profile
+
+A defined group and preferential ordering of qualities, [Custom Format](#custom-format) scores, and additional settings that can be assigned to movies, TV shows, and collections.
+
+---
+
+## Torrents
+
+### Cross-Seed
+
+The act of [seeding](#seeder) the same torrent files or data across multiple [trackers](#tracker) without re-downloading. It also refers to a [software application of the same name](https://www.cross-seed.org) that automates this process.
+
+### Deluge
+
+A lightweight, free, open-source, cross-platform BitTorrent client.
+
+### qBit Manage
+
+A program that helps manage your [qBittorrent](#qbittorrent) instance, allowing for the automation of actions on torrents based on various attributes such as time and tracker.
+
+### qBittorrent
+
+A free, open-source, cross-platform BitTorrent client.
+
+### Leecher
+
+A BitTorrent user who is actively downloading a torrent (referred to as 'leeching'). This term can also describe those who download torrents without sharing them afterward.
+
+### Peer
+
+A general term that encompasses both [Seeders](#seeder) and [Leechers](#leecher).
+
+### Seeder
+
+BitTorrent users who possess the complete set of files in a specific torrent and are sharing them, a process known as 'seeding'.
```

Comment:
add a note RE open port?

Reply:
Something like

"To actually seed content effectively, it is necessary to have some form of port-forwarding enabled. This can vary depending on whether you use a VPN or not and/or what router or firewall you have. Please consult your router manual or the documentation for your VPN or VPN container for specifics."

Link to port forwarding guide? Although I think the above is more relevant to the specific cases than the guide...

Reply:
> Something like
> 
> "To seed content effectively, it is necessary to have some form of port-forwarding enabled. This can vary depending on whether you use a VPN or not and/or what router or firewall you have. Please consult your router manual or the documentation for your VPN or VPN container for specifics."
> 
> Link to port forwarding guide? Although I think the above is more relevant to the specific cases than the guide...

I think this would be better added in some form to the port forwarding guide. When I do my pass I'll see if I can reference with a truncated version.

Reply:
In order to torrent, one of the parties must have an open incoming port. If that's not YOU, then it must be the person you're peering from. That is a small % of people, because most people are like you and don't do it.
so basically, you're getting 5-10% of possible seeders on every torrent instead of 100% of seeders on every torrent.

This is something Pearson once mentioned to explain to someone